zebrad/tests/acceptance.rs

```diff
@@ -1706,6 +1706,76 @@ async fn delete_old_databases() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn stored_config_works() -> Result<()> {
+    zebra_test::init();
+
+    // only run the test if we found a stored config
+    if let Some(stored_config_path) = stored_config_path() {
```

Comment:
This check can always skip the test. If that happens, we could break the config, and CI would still pass.

Can we require that the config is always available?
Did you want some help with the paths?

Reply:
We've had CI bugs where we always skip a test, and we never notice.
How can we make sure at least one test finds the config?

I think we can use `CARGO_MANIFEST_DIR` to find the path, it is `zebrad/Cargo.toml`:
https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

Reply:
`env!()` is what we are using and it seems to be working ok locally and in CI.